### PR TITLE
feat: improved scenarios

### PR DESCRIPTION
**Pull request by nissimsan**

### CODE REVIEW

**Review by mprorock — APPROVED** [review]

File: docs/index.html [hunk 1]
```diff
@@ -557,46 +546,26 @@ <h5>Sample E-Commerce Workflow</h5>
...
-          addition, the Bill of Lading can be linked to the Chinese manufacturer’s Commercial Invoice, Packing List, and the list
-          of DIDs associated with each set of sneakers.
-        </p>
-        <p>          
-          Our sneakers will find a home at a Danish vendor who specializes in Nike sneakers. They safely receive the container
-          from the ocean carrier and store the precious goods in a warehouse.
+            A Danish Vendor has received an online order, issues a Purchase Order (1) which is presented to the Chinese manufacturer for fulfillment (2). 
+          </p>
+        <p> 
+          The Chinese Manufacturer receives an order from their Danish customer (2). An invoice is issued (3) and presented back to customer (4).
```
Comment (TallTed) [inline docs/index.html:552]:
```suggestion
A Danish Vendor has received an online order, issues a Purchase Order (1)
which is presented to the Chinese manufacturer for fulfillment (2). 
          </p>
        <p> 
The Chinese Manufacturer receives an order from their Danish customer (2).
An invoice is issued (3) and presented back to customer (4). 
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Mostly whitespace to improve source readability. Some punctuation and the like.

**Review by OR13 — APPROVED** [review]
I am ok with these changes @nissimsan please accept for reject @TallTed 's suggestions.

**Comment by nissimsan** [comment]
Thanks, @TallTed - keeping me on the straight'n'narrow! :)

**Pull request merged**